lib/card.rb

```diff
@@ -0,0 +1,18 @@
+class Card
+
+  attr_reader :id
+
+  def initialize(id, attributes)
+    @id = id
+    @attributes = attributes
+  end
+
+  def method_missing(method_name, *args)
```

Comment:
if you choose to keep the `method_missing` override, you should also override `respond_to?`